src/metrics/misc.ts

```diff
@@ -221,13 +263,36 @@ export class MetadataMetric implements Metric {
     this.mostGamesPlayed = maxGames;
     this.playerMostGames = playerMostGames;
 
+    // sort gameTypeStats from greatest to least by number of times they appear
+    let sortedGameTypeStats = Object.entries(this.gameTypeStats).sort((a, b) => b[1] - a[1]);
+    let sortedGameTypeStatsObj = Object.fromEntries(sortedGameTypeStats);
+    this.gameTypeStats = sortedGameTypeStatsObj as {
+      numberUltraBulletGames: number;
+      numberBulletGames: number;
+      numberBlitzGames: number;
+      numberRapidGames: number;
+      numberClassicalGames: number;
+      numberOtherGames: number;
+    };
+    
+    // sort gameTimeControlStats from greatest to least by number of times they appear
+    let sortedGameTimeControlStats = Object.entries(this.gameTimeControlStats).sort((a, b) => b[1] - a[1]);
+    let sortedGameTimeControlStatsObj = Object.fromEntries(sortedGameTimeControlStats);
+    this.gameTimeControlStats = sortedGameTimeControlStatsObj;
+
+    // sort the openings from greatest to least by number of times they appear
+    let sortedOpenings = Object.entries(this.openings).sort((a, b) => b[1].appearances - a[1].appearances);
+    let sortedOpeningsObj = Object.fromEntries(sortedOpenings);
+    this.openings = sortedOpeningsObj;
```

Comment:
I'm confused what we are doing here. We are sorting the objects why? And separately
- We should prefer const when a var is not reassigned
- We should not create variables just to immediately assign them to something else.

Reply:
sorting bc prints to console nicer and gives me the facts the way i want them. feel free to suggest a change that still gives me the information in this table: https://discord.com/channels/1156826703440646145/1157595718484250714/1173198811460554812

good note re: const

why not create variables for reassignment? does creating variables a computationally heavy thing to do relatively speaking?